Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN npm run build:dist --prefix web
 ## Build final container
 FROM gcr.io/distroless/base
 
+ENV SERVER_URL localhost
```

Comment:
sorry for nitpicking but I think that "url" is not the right term in this context, since we're not passing the protocol. Furthermore, the protocol is hardcoded to http somewhere else in the codebase. 
To me, we should either rename the variable from SERVER_URL to SERVER_HOST or include the protocol in the url as well.

WDYT?

Reply:
Agree